target/riscv/cheri-archspecific-early.h

```diff
@@ -84,11 +84,13 @@ enum CheriSCR {
     CheriSCR_UTDC = 5,
     CheriSCR_UScratchC = 6,
     CheriSCR_UEPCC = 7,
+    CheriSCR_UTIDC = 8,
```

Comment:
It doesn't really matter too much since this is a prototype but the allocation of numbers was supposed to be 0-7 for U Mode, 8-15 for S, etc. so 3 and 11 would make more sense.

Reply:
You are right, thanks! I will change it to the numbers you proposed.

On a different note, all U mode SCRs are based on the N extension, which has vanished quite some time ago...